apps/web/components/eventtype/EventAvailabilityTab.tsx

```diff
@@ -160,10 +154,11 @@ EventTypeScheduleDetails.displayName = "EventTypeScheduleDetails";
 
 const EventTypeSchedule = ({ eventType }: { eventType: EventTypeSetup }) => {
   const { t } = useLocale();
+  const formMethods = useFormContext<FormValues>();
   const { shouldLockIndicator, isManagedEventType, isChildrenManagedEventType } = useLockedFieldsManager(
     eventType,
-    t("locked_fields_admin_description"),
-    t("locked_fields_member_description")
+    formMethods,
```

Comment:
We now need form methods in order to set discretely the different fields in the event type metadata